tests/portability/cmsis_rtos_v1/src/kernel_apis.c

```diff
@@ -19,7 +19,7 @@
  * Additionally, if RC is used for low frequency clock then it has 5% tolerance.
  */
 #define TOLERANCE_PPC \
-	1 + 8 + (IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC) ? 5 : 0)
+	(1 + 8 + (IS_ENABLED(CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC) ? 5 : 0))
```

Comment:
As a side note, the comment describing this macro does not explain this leading 1 which actually seems to be not needed (currently the used tolerance is 9 or 14%, from the comment I'd assume it would be 8 or 13%). Maybe this could be corrected on the occasion?

Reply:
this 1 is a "base" to cover cpu processing in the test.
```
start_time = osKernelSysTick();
k_busy_wait(WAIT_TIME_US);
stop_time = osKernelSysTick();
```
I've added comment in the common section.